config/packages/lock.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+
+    app.lock.store:
+        class: Symfony\Component\Lock\Store\PdoStore
+        factory: [Symfony\Component\Lock\Store\StoreFactory,  createStore]
+        arguments: ['%env(DATABASE_URL)%']
+
+    app.lock.blocking_store:
```

Comment:
not needed with symfony/lock 5.2

Reply:
I'll have to update then =)

Thank you